src/pages/checkout.tsx

```diff
@@ -60,23 +91,43 @@ export default function Checkout(props: CheckoutPageProps) {
 
   const hasValidAddress = () => {
     const vals = [
-      address.name,
+      address.firstName,
+      address.lastName,
       address.email,
       address.address,
       address.city,
       address.state,
       address.zip,
     ];
+    if(!address.email.includes("@") || !address.email.includes(".")) return false;
```

Comment:
can you put brackets around this if statement please, imo it's just better style and will avoid formatting causing an actual bug later